initialize.sh

```diff
@@ -20,6 +20,12 @@ check_for_preseeds() {
   fi
 }
 
+if [ "$RAILS_ENV" = "production" ]
+then
+    echo "This script is not intended for usage with RAILs_ENV=production. Aborting."
+    exit 1
+fi
+
```

Comment:
The file is actually not used by the production environment as far as I can see. Please verify this.

Reply:
Checked it and can confirm.